cumulusci/core/config/tests/test_config_expensive.py

```
@@ -444,6 +448,94 @@ def test_get_access_token__no_access_token(self, Command):
+            ),
+            mock.Mock(
+                stderr=io.BytesIO(b""),
+                stdout=io.BytesIO(show_password),
+                returncode=0,
+            ),
+        ]
+
+        config = SfdxOrgConfig({"username": "test", "created": True}, "test")
+        info = config.sfdx_info
+
+        assert info["password"] == "s3cr3t-pw"
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "password"

(S105)